.ci/pipeline/build-and-test-lnx.yml

```diff
@@ -48,7 +45,7 @@ steps:
       bash .ci/scripts/setup_sklearn.sh $(SKLEARN_VERSION)
       pip install --upgrade -r requirements-test.txt
       pip install $(python .ci/scripts/get_compatible_scipy_version.py)
-      if [ $(echo $(PYTHON_VERSION) | grep '3.9\|3.11') ] && [ $(SKLEARN_VERSION) != "1.0" ]; then conda install -q -y -c https://software.repos.intel.com/python/conda/ dpctl=0.17.0 dpnp=0.15.0; fi
+      if [ $(SKLEARN_VERSION) != "1.0" ]; then pip install dpctl==0.18.* dpnp==0.16.*; fi

```

Comment:
Are we installing dpctl/dpnp for all py versions?

Reply:
Yes, it has same supported python versions as sklearnex.

Reply:
that does, however, cause certain functionality within sklearnex to become untested (see DummySyclQueue)

Reply:
Is DummySyclQueue needed if dpctl supports all sklearnex versions?

Reply:
Are there circumstances at this point where dpctl would not be available?

Reply:
I can give at least two examples:
2. Installation Issues: problems on the user’s side during the installation of dpctl.
3. Environment Compatibility: for example, dpctl does not currently support Python 3.13. While scikit-learn already supports Python 3.13 and we may soon add it to our matrices, dpctl will not be available for this Python version at this time.